tests/AutoReview/FixerFactoryTest.php

```diff
@@ -152,7 +151,6 @@ public function provideFixersPriorityCases()
             [$fixers['php_unit_fqcn_annotation'], $fixers['php_unit_ordered_covers']],
             [$fixers['php_unit_no_expectation_annotation'], $fixers['no_empty_phpdoc']],
             [$fixers['php_unit_no_expectation_annotation'], $fixers['php_unit_expectation']],
-            [$fixers['php_unit_strict'], $fixers['php_unit_construct']],
```

Comment:
there is no prio issue here, both fixers deal with `assertEquals` and `assertSame` and v.v.